README.md

```diff
@@ -2,17 +2,16 @@
 
 
 # WHAT? 
-We are creating an application to drive the car maunally for seeking treasure(s). The car will be able to take multiple routes to find the treasure. The environment will consist of multiple objects/obstacles and one treasure. The user will be alerted when there is an object nearby and the car will be stopped if it gets too close to the object. The closer or further the car is from the treasure will be displayed on the app. The user will be able to view the car's speed and how much the car has travelled so far. The user will have a first person view (driver seat view and rear view). 
+We are creating an application to drive the car manually using a joystick. The car will be able to travel freely around the map. The user will be alerted when there is an object nearby through a pop up message on the app. If the car gets too close, the emergency break will be activated and the car will stop moving. The user will be able to view the car's speed and how much the car has travelled so far. The user will have a first person view (driver seat view and rear view).  
```

Comment:
You cannot have **two** cameras

Reply:
Is it possible to switch views then? I.e not having two camera views at the same time but choosing which one should be activated and switch between them.

Reply:
No